NEWS.md

```diff
@@ -63,6 +63,7 @@ PR: #2638 (@pfistl)
 
 - Bugfix: Allow `method = "vh"` for filter `randomForestSRC_var.select` and return informative error message for not supported values. Also argument `conservative` can now be passed. See #2646 and #2639 for more information (@pat-s, #2649)
 * Bugfix: With the new _praznik_ v7.0.0 release filter `praznik_CMIM` does no longer return a result for logical features. See https://gitlab.com/mbq/praznik/issues/19 for more information
+- Allow a custom threholding function to be passed to filterFeatures and makeFilterWrapper (#2686)
```

Comment:
This news item is also not at the correct position. Please move it below "development version".

Reply:
OK. There was no "filters - general" section under the development version so I have created one. Is that OK?

Reply:
Yes, that was what I meant.